Canvas/Canvas/CanvasAppDelegate+Helm.swift

```diff
@@ -37,7 +37,11 @@ extension AppDelegate: RCTBridgeDelegate {
 
         HelmManager.shared.onReactReload = {
             guard self.window?.rootViewController is CanvasTabBarController else { return }
-            self.changeUser()
+            guard let session = Keychain.mostRecentSession else {
+                self.changeUser()
+                return	
+            }
+            self.setup(session: session, wasReload: ProcessInfo.processInfo.arguments.contains("-Reload"))
```

Comment:
I'm not sure what the reason was for doing `changeUser` instead of taking us right back into the same logged in user on reload so if this is incorrect let me know.

Reply:
I think this is a good change. The important thing was making sure that reset the root view controller.

I'm not sure I follow what's going on with the command line argument though. Wouldn't we always pass true from here?

Reply:
I've switched to a launch argument. If the launch argument is enabled this will return true, if not it will return false.

Reply:
Is that so we'll only get the js back to my spot if you opt in?

Reply:
Correct. CMD-Option-R will open launch arguments and you can toggle it.

Reply:
So why not just always do that? I'd rather not have more options around. Reload is already a dev feature.